src/inputs/text-with-button.jsx

```diff
@@ -15,6 +15,7 @@ export function TextInputWithButton({ name, label, buttonLabel, onButtonClick, l
       <div className="field has-addons">
         <div className="control">
           <input
+            data-testid={testid}
```

Comment:
`name` is unique here because it maps to a formik input label. We can use that as our test id

```
data-testid={`text-input-${name}`}
```